docs/graphql/manual/api-reference/graphql-api/index.rst

```diff
@@ -28,9 +28,60 @@ The following types of requests can be made using the GraphQL API:
 - :doc:`Query / Subscription <query>`
 - :doc:`Mutation <mutation>`
 
+Batching requests
```

Comment:
Well it's not just queries, because you can mix queries and mutations, so I need a word that covers both of those. Perhaps "request" isn't ideal since there is also "the request", but just saying "query" would be incomplete.

Reply:
I switched it to "operations" since that's the word the GraphQL spec uses.